src/knuto/kafka_user_topic.py

```diff
@@ -29,8 +29,13 @@ def log_and_raise(msg):
         if resource["patternType"] not in ["literal", "prefix"]:
             log_and_raise(f"Unsupported patternType {resource['patternType']}, operator needs upgrade?")
 
-        if operation == "Write" and not resource["name"].startswith(f"{namespace}-"):
-            log_and_raise(f"ACL {idx}: resource name {resource['name']} does not begin with {namespace}-, operation Write not allowed")
+        if operation == "Write" and (
+                not resource["name"].startswith(f"{namespace}-") and
+                resource["name"] not in globalconf.allowed_non_namespaced_topics):
+            log_and_raise(f"ACL {idx}: resource name {resource['name']} does "
+                          f"not begin with {namespace}- or are included in "
```

Comment:
I think a better message is: "does neither begin with {namespace}- nor is it included in"